beacon/sync/src/main/java/tech/pegasys/teku/beacon/sync/fetch/AbstractFetchTask.java

```diff
@@ -26,7 +26,11 @@
 import tech.pegasys.teku.networking.p2p.network.P2PNetwork;
 import tech.pegasys.teku.networking.p2p.peer.NodeId;
 
-public abstract class AbstractFetchTask<T> {
+/**
+ * @param <K> - the key used to fetch
```

Comment:
Would be clearer, I think
```suggestion
 * @param <K> - the key type used to fetch
```